src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -175,6 +177,54 @@ private void AddExceptions(ISymbol symbol)
                 }
             }
 
+            /// <summary>
+            /// If the symbol is a local or anonymous function (lambda or delegate), adds the variables captured
+            /// by that local or anonymous function to the "Captures" group.
+            /// </summary>
+            /// <param name="symbol"></param>
+            protected abstract void AddCaptures(ISymbol symbol);
+
+            /// <summary>
+            /// Given the body of a local or an anonymous function (lambda or delegate), add the variables captured
+            /// by that local or anonymous function to the "Captures" group.
+            /// </summary>
+            protected void AddCaptures(SyntaxNode syntax)
+            {
+                var semanticModel = GetSemanticModel(syntax.SyntaxTree);
+                if(semanticModel.IsSpeculativeSemanticModel)
+                {
+                    // In the context of symbol completion, it is possible we'll make a description for a symbol while speculating (but it won't be displayed)
```

Comment:
lolwut?  we make a description, but don't display it?  

Also, if that's the case, why add the group at all?  why not just insta bail?

Finally, if you are going to check IsSpeculativeSemanticModel, it's worth calling that that's because data flow doens't work with SpeculativeSemanticModels, so you need to avoid the code below this check.

Reply:
I'll clarify the comment. When you complete a local symbol, we make a description (with many parts: main description, exceptions, captures, etc), but only the main description seems to be displayed.
The reason I put captures here anyways, is in case I'm wrong and missed some scenario. Captures listing "?" is less ambiguous/misleading than displaying no captures at all.

Thanks!